doc/source/image_description/elements.rst

```diff
@@ -544,6 +544,11 @@ target_removable="true|false":
   of the target machine. By default the target disk is
   expected to be non-removable
 
+selinux_policy.attribute="targeted|mls":
```

Comment:
There are three types: `targeted`, `mls`, and `minimum`. The default should be `targeted`.

Reply:
Thanks, added it to the schema